src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/msquic_generated.cs

```diff
@@ -0,0 +1,2677 @@
+                public ulong AbsoluteOffset;
+
+                [NativeTypeName("uint64_t")]
+                public ulong TotalBufferLength;
+
+                [NativeTypeName("const QUIC_BUFFER *")]
+                public QUIC_BUFFER* Buffers;
+
+                [NativeTypeName("uint32_t")]
+                public uint BufferCount;
+
+                public QUIC_RECEIVE_FLAGS Flags;
+            }
+
+            internal unsafe partial struct _SEND_COMPLETE_e__Struct
+            {
+                [NativeTypeName("BOOLEAN")]
+                public byte Canceled;
+
+                public void* ClientContext;
+            }
+
+            internal partial struct _PEER_SEND_ABORTED_e__Struct
+            {
+                [NativeTypeName("QUIC_UINT62")]
+                public ulong ErrorCode;
+            }
+
+            internal partial struct _PEER_RECEIVE_ABORTED_e__Struct
+            {
+                [NativeTypeName("QUIC_UINT62")]
+                public ulong ErrorCode;
+            }
+
+            internal partial struct _SEND_SHUTDOWN_COMPLETE_e__Struct
+            {
+                [NativeTypeName("BOOLEAN")]
+                public byte Graceful;
+            }
+
+            internal partial struct _SHUTDOWN_COMPLETE_e__Struct
+            {
+                [NativeTypeName("BOOLEAN")]
+                public byte ConnectionShutdown;
+
+                public byte _bitfield;
+
+                [NativeTypeName("BOOLEAN : 1")]
+                public byte AppCloseInProgress
+                {
+                    get
+                    {
+                        return (byte)(_bitfield & 0x1u);
+                    }
+
+                    set
+                    {
+                        _bitfield = (byte)((_bitfield & ~0x1u) | (value & 0x1u));
+                    }
+                }
+
+                [NativeTypeName("BOOLEAN : 7")]
+                public byte RESERVED
+                {
+                    get
+                    {
+                        return (byte)((_bitfield >> 1) & 0x7Fu);
+                    }
+
+                    set
+                    {
+                        _bitfield = (byte)((_bitfield & ~(0x7Fu << 1)) | ((value & 0x7Fu) << 1));
+                    }
+                }
+            }
+
+            internal partial struct _IDEAL_SEND_BUFFER_SIZE_e__Struct
+            {
+                [NativeTypeName("uint64_t")]
+                public ulong ByteCount;
+            }
+        }
+    }
+
+    internal unsafe partial struct QUIC_API_TABLE
+    {
+        [NativeTypeName("QUIC_SET_CONTEXT_FN")]
+        public delegate* unmanaged[Cdecl]<QUIC_HANDLE*, void*, void> SetContext;
+
+        [NativeTypeName("QUIC_GET_CONTEXT_FN")]
+        public delegate* unmanaged[Cdecl]<QUIC_HANDLE*, void*> GetContext;
+
+        [NativeTypeName("QUIC_SET_CALLBACK_HANDLER_FN")]
+        public delegate* unmanaged[Cdecl]<QUIC_HANDLE*, void*, void*, void> SetCallbackHandler;
```

Comment:
For MsQuic: Other functions accepting a callback accept a strongly typed function pointer, can we have it here as well?